vispy/glsl/build-spatial-filters.py

```diff
@@ -654,9 +655,37 @@ def weight(self, x):
 print("")
 print("const float kernel_bias  = %f;" % bias)
 print("const float kernel_scale = %f;" % scale)
+print("const float kernel_size = %f;" % n)
 print("uniform sampler2D u_kernel;")
 print("")
 
+code = 'float\n'
+code += 'unpack(vec4 rgba)\n'
+code += '{\n'
+code += '\t// return rgba.r;  // uncomment this for r32f debugging\n'
+code += '\trgba.rgba = rgba.abgr * 255;\n'
```

Comment:
The one thing I don't get is why the `rgba.abgr` above gets multiplied by 255. That makes me think the maximum value should be 255 + 255 / 256. + 255 / (256 * 256) + 255 / (256 * 256 * 256), which is about 256. Maybe the above `rgba.rgba = rgba.abgr` shouldn't be multiplied by 255?